x-pack/metricbeat/module/enterprisesearch/health/_meta/data.json

```diff
@@ -0,0 +1,45 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "enterprisesearch": {
+        "health": {
+            "crawler": {
+                "workers": {
+                    "active": 0,
+                    "available": 16,
+                    "pool_size": 16
+                }
+            },
+            "jvm": {
+                "memory_usage": {
+                    "heap_committed.bytes": 1846018048,
```

Comment:
I'm not trying to be pedantic but it's preferred to have something like
```json
"heap": {
  "committed": {
    "bytes": 1846018048
  },
  "init": {
    "bytes": 2147483648
  }
}
```

Check `elasticsearc/cluster_stats/data.json` to see an example. We also don't generate JSON's with dots on their key fields.

Reply:
Thank you for pointing this out. I had no idea what was the preferred way of naming those metrics, so went with dot-suffixes. Going update it all ASAP.